pkg/virt-v2v/customize/scripts/windows/9999-restore_config-legacy.ps1

```diff
@@ -0,0 +1,46 @@
+# Migration - Reconfigure disks
+# A legacy version of the script that does not use Get-Disk
+
+Write-Host "`nRe-enabling all offline drives`n"
+
+# Find VirtIO disks and their disk numbers
+$virtioDisks = Get-WmiObject -Class Win32_DiskDrive | Where-Object {
+    $_.Model -like '*VirtIO*' -or $_.Caption -like '*VirtIO*'
+}
+
+if ($virtioDisks) {
+    foreach ($disk in $virtioDisks) {
+        Write-Host "Processing disk $($disk.Index): $($disk.Model)`n"
+
+        # Prepare diskpart script content
+        $diskpartScriptContent = @"
+san policy=OnlineAll
```

Comment:
This will change the san policy for newer disks.
This would be security risk, please remove it.